### PR TITLE
Only load prism css on guides instead of all pages at gruntwork.io

### PR DESCRIPTION
**Pull request by Etiene**
1. does not use same layout of default.html for post.html
2. copy pastes contents of default.html around post.html
3. removes prism.css from head 
4. adds prism.css to post.html

### CODE REVIEW

File: _layouts/post.html [hunk 1]
```diff
@@ -1,49 +1,59 @@
...
-          </div>
+<!doctype html>
+<html lang="en">
+    <head>
+        {% include head.html %}
+        <link rel="stylesheet" href="{{ '/assets/css/prism.css' | prepend: site.baseurl }}"/>
+    </head>
+    <body{% if page.slug %} class="{{ page.slug }}{% if page.slug != 'index-page' and page.slug !='how-it-works' and page.slug != 'guides' %} sub-page{% endif %}" {% endif %}>
+        {% include navbar.html %}
+        <div  class="post-bg">
```
Comment (Etiene) [inline _layouts/post.html]: tab

**Comment by Etiene** [comment]
> What's the advantage of duplicating the layout code?

So `post` and `default` can have different `<head>` contents, one with and one without prism. Before there was one single layout, `default`, with prism.

**Comment by brikis98** [comment]
> So post and default can have different <head> contents, one with and one without prism. Before there was one single layout, default, with prism.

Can we make CSS conditional, like JS was? So, one layout, but an if-statement/loop to add custom CSS on some pages?

**Comment by brikis98** [comment]
Just noticed this PR was never merged?

**Comment by mcalhoun** [comment]
closing stale PR

**Pull request closed without merging**